src/custom/components/swap/EthFlow/containers/EthFlowModal/hooks/useRemainingNativeTxsAndCosts.ts

```diff
@@ -0,0 +1,99 @@
+import { useMemo } from 'react'
+import { CurrencyAmount, Currency } from '@uniswap/sdk-core'
+import { useGasPrices } from 'state/gas/hooks'
+import { useWeb3React } from '@web3-react/core'
+import { BalanceChecks } from '../../../pure/EthFlowModalContent/EthFlowModalTopContent'
+
+import { BigNumber } from '@ethersproject/bignumber'
+import { AVG_APPROVE_COST_GWEI } from 'constants/index'
+import { parseUnits } from 'ethers/lib/utils'
+// eslint-disable-next-line no-restricted-imports
+
+export const MINIMUM_TXS = '10'
+export const DEFAULT_GAS_FEE = parseUnits('50', 'gwei')
+
+export function _estimateTxCost(gasPrice: ReturnType<typeof useGasPrices>, native: Currency | undefined) {
+  if (!native) {
+    return {}
+  }
+  // TODO: should use DEFAULT_GAS_FEE from backup source
+  // when/if implemented
+  const gas = gasPrice?.standard || DEFAULT_GAS_FEE
+
+  const amount = BigNumber.from(gas).mul(MINIMUM_TXS).mul(AVG_APPROVE_COST_GWEI)
```

Comment:
Probably not for this PR but I feel like some comment with explanation would be useful here.